src/dlls/dbgshim/dbgshim.cpp

```diff
@@ -1052,6 +1053,56 @@ IsCoreClrWithGoodHeader(
     return false;
 }
 
+static
+HMODULE*
+EnumProcessModulesInternal(
+    HANDLE hProcess, 
+    DWORD *pCountModules)
+{
+    *pCountModules = 0;
+
+    // Start with 1024 modules
+    DWORD cbNeeded = sizeof(HMODULE) * 1024;
+
+    ArrayHolder<BYTE> modules = new (nothrow) BYTE[cbNeeded];
```

Comment:
Do you know if the CRT guarantees that the result of `new BYTE[]` is pointer-size alligned?

Reply:
I don't know for sure.  That wouldn't be a problem on x86/x64 other than perf problems, but it might cause problems on ARM.